kinvey.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2017, Kinvey, Inc. All rights reserved.
+ *
+ * This software is licensed to you under the Kinvey terms of service located at
+ * http://www.kinvey.com/terms-of-use. By downloading, accessing and/or using this
+ * software, you hereby accept such terms of service  (and any agreement referenced
+ * therein) and agree that you have read, understand and agree to be bound by such
+ * terms of service and are of legal age to agree to such terms with Kinvey.
+ *
+ * This software contains valuable confidential and proprietary information of
+ * KINVEY, INC and is subject to applicable licensing agreements.
+ * Unauthorized reproduction, transmission or distribution of this file and its
+ * contents is a violation of applicable laws.
+ */
+
+const path = require('path');
+// TODO: Figure out if we really need this - seems it's not bad to stay, just remove max upload size
+process.env.NODE_CONFIG_DIR = path.join(__dirname, 'config');
+
+const updateNotifier = require('update-notifier');
+
+const config = require('config');
+const CLIManager = require('./lib/cli-manager');
```

Comment:
Filenames should reflect the name of the default export (https://github.com/Kinvey/style-guides/blob/master/backend/es6.md#naming--filename-matches-export)

Applies to any other instances of the same